Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent() {
```

Comment:
Try to use more arrow functions when creating functional components `const CardContent = props => {}`